pkg/crc/constants/constants.go

```diff
@@ -128,20 +128,9 @@ func BinDir() string {
 
 // GetHomeDir returns the home directory for the current user
 func GetHomeDir() string {
-	if runtime.GOOS == "windows" {
-		if homeDrive, homePath := os.Getenv("HOMEDRIVE"), os.Getenv("HOMEPATH"); len(homeDrive) > 0 && len(homePath) > 0 {
-			homeDir := filepath.Join(homeDrive, homePath)
-			if _, err := os.Stat(homeDir); err == nil {
-				return homeDir
-			}
-		}
-		if userProfile := os.Getenv("USERPROFILE"); len(userProfile) > 0 {
-			if _, err := os.Stat(userProfile); err == nil {
-				return userProfile
-			}
-		}
-	}
-	return os.Getenv("HOME")
+	homeDir, _ := os.UserHomeDir()
```

Comment:
@praveenkumar @gbraad some thoughts here ? Maybe you have some history?

Should the code panic in case of an error ?

Reply:
@guillaumerose Without this patch we don't handle error here, but may be we should and panic, which can be part of another commit, since this commit is not changing the existing flow of code.